packages/trpc/server/routers/viewer/organizations/getTeams.handler.ts

```diff
@@ -16,10 +15,6 @@ export async function getTeamsHandler({ ctx }: GetTeamsHandler) {
 
   if (!currentUser.organizationId) throw new TRPCError({ code: "UNAUTHORIZED" });
 
-  // check if user is admin of organization
-  if (!(await isOrganisationAdmin(currentUser?.id, currentUser.organizationId)))
-    throw new TRPCError({ code: "UNAUTHORIZED" });
-
```

Comment:
@Amit91848 why did you remove this?

Reply:
I'll have to confirm if we allow non admins to get the teams

Reply:
In order for members to filter based on teams within an organization, they would need access to view those teams. 

> I'll have to confirm if we have to allow non admins to get the teams

Will change it back to only admins being able to access and use team filters in this case.